json-schemas/src/client-events-api-requests.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably API Requests Client Event",
+  "type": "object",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "the host the request was made to"
```

Comment:
Let's format these descriptions as proper sentences (i.e. start with a capital letter, and end with a full stop).

Also, I assume this is the HTTP Host header from the request, in which case I'd alter the description to:

```
The HTTP Host header of the API request.
```

Reply:
> Let's format these descriptions as proper sentences (i.e. start with a capital letter, and end with a full stop).

For some reason I was convinced that the convention for JSON Schema was to drop both, but I cannot find any evidence of that now. 

> Also, I assume this is the HTTP Host header from the request, in which case I'd alter the description to:

Could be, I'll check.

Reply:
My reasoning is that I expect these descriptions to appear in both documentation (e.g. [here](https://ably.com/documentation/realtime/metachannels#connections) and in UIs when users interact with these messages, where I would expect them to be proper sentences.